docs/en/enterprise-edition/content-collections/administration/configure-external-integrations-on-prisma-cloud/integrate-prisma-cloud-with-servicenow.adoc

```diff
@@ -264,3 +353,29 @@ image::administration/servicenow-security-incidents-alerts.png[]
 .. To view event incidents (events table), select *Event Management > All Events*.
 +
 image::administration/servicenow-event-incidents-alerts.png[]
+
+
+[#manage-snow-integration]
+=== Manage your ServiceNow integration
+
+If you have an existing ServiceNow Integration on Prisma Cloud, you can view, edit, or delete an integration from the *Actions* panel on the *Integrations & Notifications* page. The *Actions* panel provides options to manage your integrations and notifications on Prisma Cloud.
+
+image::administration/manage-snow-integration.png[]
+
+* *View—* Select the *View* Integration icon on the *Actions* pane. The View ServiceNow Integration page will appear, displaying the current configuration details of the ServiceNow integration.
+
+* *Edit—* Select the *Edit* icon on the *Actions* pane. The Edit ServiceNow Integration page will appear, allowing you to modify relevant configuration details such as enabling Oauth 2.0 Authentication or updating Refresh Token details.
+
+** To enable *Oauth 2.0 Authentication*, perform the following steps:
+
+*** Enter the *Client ID* that you had copied earlier from the ServiceNow instance.
+
+*** Enter the *Client Secret* that you had copied earlier from the ServiceNow instance.
+
+*** Enter the *Refresh Token* that you had copied from the response in Postman.
++
+Once the *Refresh Token* expires, you must perform the step 5—xref:integrate-prisma-cloud-with-servicenow.adoc#idce37e68b-d094-4b6b-a5d4-ab21d092fd36[Generate Refresh Token] and include it in the Prisma Cloud UI. For example, if you set the token value as 8,640,000 seconds (100 days), then after 100 days, you must reauthorize the OAuth connection.
```

Comment:
I'm not clear on what step 5 is here. When I click the hyperlink it does not take me to step 5 and the list above has only 4 steps. 
![image](https://github.com/hlxsites/prisma-cloud-docs/assets/110043374/1e8f1159-7d03-4860-858d-47188cd2777c)

Reply:
ok. I will re-write the sentence for more clarity. The hyperlink will not take you to step 5 because of our infrastructure limitations. There is no way to hyperlink between steps in a task. This functionality was supported pre-darwin.